corehq/apps/accounting/bootstrap/config/testing.py

```diff
@@ -7,15 +7,15 @@
 )
 
 BOOTSTRAP_CONFIG_TESTING = {
-    (SoftwarePlanEdition.COMMUNITY, False, False): {
+    (SoftwarePlanEdition.COMMUNITY, False, False, False): {
```

Comment:
I guess the test won't fail if you didn't update those configs since you have made changes to `ensure_plans` to account for old config file who don't have `is_annual_plan`. Can you validate it ?( by checking out a previous commit and run the test)

Reply:
I found it was failing, and I think that once `DefaultProductPlan` has `is_annual_plan` property on it, added to its `unique_together` constraint, and objects created using `is_annual_plan=True`, that key should always be passed in. 

The reason being that in `ensure_plans`, we try to get a `DefaultProductPlan` directly based on whether or not that key exists. If we leave out that key at this point, `get()` can return multiple objects (one with `is_annual_plan=True` and one `is_annual_plan=False`), which throws an unexpected error. 
https://github.com/dimagi/commcare-hq/blob/486bd7811aa2362f2d79852e8ca47dbc65aeeef0/corehq/apps/accounting/bootstrap/utils.py#L217-L221

So, essentially that key is optional in the past, but is required going forward.